Makefile

```diff
@@ -99,3 +99,7 @@ validate-no-changes: compile
 		echo "dist directory is not clean. run 'make build'"; \
 		exit 1; \
 	fi
+
+.PHONY: npm-publish
+npm-publish:
+	npm publish --tag alpha --tag latest --access public
```

Comment:
learned.

Reply:
btw do you need any credentials?

Reply:
Yeah, you need credentials. Would need to create an account at npmjs.org and get added to our organization.